Snowflake.Data.Tests/SFBaseTest.cs

```diff
@@ -176,8 +176,10 @@ public void CreateTestTimeArtifact()
             
             var dotnetVersion = Environment.GetEnvironmentVariable("net_version");
             var cloudEnv = Environment.GetEnvironmentVariable("snowflake_cloud_env");
+
+            var separator = Path.DirectorySeparatorChar;
             
-            File.WriteAllText($"../../../{GetOs()}_{dotnetVersion}_{cloudEnv}_performance.csv", resultText);
+            File.WriteAllText($"..{separator}..{separator}..{separator}{GetOs()}_{dotnetVersion}_{cloudEnv}_performance.csv", resultText);

```

Comment:
maybe let's add a comment why do we need to go three times down the tree?

Reply:
Done